### PR TITLE
Fix and improve video width selection

### PR DESCRIPTION
**Pull request by rowanbeentje**
There's currently code in `get-rendition` to select which size of video to use, which was documented as being controlled by the `optimumwidth` video parameter.  However... the rendition code didn't use any supplied width, possibly due to [this change in n-video](https://github.com/Financial-Times/n-video/pull/36).  As a result the live behaviour is that the documentation is wrong and the supplied `optimumwidth` parameter is only used for the placeholder image.

Rather than breaking the functionality for anyone currently depending on the size behaviour, I've added a new option for `optimumvideowidth` to allow rendition selection.

The rendition selection code was also a bit broken; it sorted the sizes, then chose the size which was closest in width to the size supplied, as long as it had a larger video to select.  If there were no videos larger than the optimum size selected, it instead chose the _smallest_ video; I've changed this to select the largest.

### CODE REVIEW

File: src/js/helpers/get-rendition.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ function getRendition(renditions, options) {
 		return false;
 	});
 
-	return appropriateRendition || orderedRenditions.shift();
+	return appropriateRendition || orderedRenditions.pop();
```
Comment (JakeChampion) [inline src/js/helpers/get-rendition.js:31]: 👍

Comment (JakeChampion) [inline src/js/helpers/get-rendition.js:31]: Can we have a test for this functionality?

Comment (JakeChampion) [inline src/js/helpers/get-rendition.js:31]: You added one already, great work! (https://github.com/Financial-Times/o-video/pull/29/files#diff-fb856e7d307df4c5b3a7f1aa31c40683R29)

**Comment by JakeChampion** [comment]
LGTM, nice work on this @rowanbeentje 👍

**Comment by AlbertoElias** [comment]
Looks good!

**Pull request merged**